api/src/app.ts

```diff
@@ -187,8 +187,7 @@ export const build = async (
     }
   });
 
-  // NOTE: Code behind the `FREECODECAMP_NODE_ENV` var is not ready to be deployed yet.
-  if (FREECODECAMP_NODE_ENV !== 'production') {
+  if (FCC_ENABLE_EXAM_ENVIRONMENT) {
```

Comment:
```suggestion
  if (FCC_ENABLE_EXAM_ENVIRONMENT === "true") {
```

Reply:
It can't be `"true"`, since env converts it to `true`. Booleans are easier to work with, imo.

Reply:
I did completely fail to update the tests, mind. Fixed now.